### ISSUE TITLE
Improve “Writing your own components” section

### ISSUE THREAD

**Issue by valentijnnieman** [plotly/dash-docs#106]
When following the tutorial on how to write your own components for Dash, I came across some stuff that I thought could be improved:

* When trying out my custom component in a “real” project (with `python setup.py install`), when running the app, Dash tries to download the package from unpkg, but because I hadn't done step 7 yet which handles publishing a component, it couldn't find it and it threw some errors. A workaround could be to use `app.scripts.config.serve_locally = True` in the Dash app, I think it’s good to add this to the tutorial (maybe in step 5 or 6)
* I was stuck for a bit when trying to add styling to my custom component - I talked to @nicolaskruchten about this and he made the point that this encourages experimentation with CSS and the like, which is a good point, but I still think some guidance here could be beneficial. I suggest adding maybe a couple of lines that explain how it works in `dash-core-components`, and/or maybe some suggestions of how it could also work: inline styles, css-in-js, styled-jsx, glamorous, etc, and the suggestion for the user to experiment with it. I guess my point is that I felt I was missing something, that I was just missing a line somewhere that talked about how to handle styling, and that confused me. 

That’s all I can think of for now, they’re not big improvements, but I think it will help to make the tutorial a bit more welcoming to new users! 

Let me know what you think, @chriddyp !

**Comment by chriddyp** [comment]
Yeah, love it! Feel free to make a PR into this repo. Also, it seems like `app.scripts.config.serve_locally = True` and `app.css.config.serve_locally = True` needs to be set in the default `usage.py` file that we include in the repo (https://github.com/plotly/dash-components-archetype/blob/master/init/usage.py#L7) and that we need to include some more info in the plugin chapter that they should start with `usage.py`.

I would 👍 a PR that improved this chapter with these suggestions
